frontend/src/app/[locale]/process/ProcessProgress.tsx

```diff
@@ -0,0 +1,79 @@
+import { ExternalRoutes } from "src/constants/routes";
+
+import { useMessages, useTranslations } from "next-intl";
+import Link from "next/link";
+import React from "react";
+import {
+  Grid,
+  GridContainer,
+  IconList,
+  IconListContent,
+  IconListIcon,
+  IconListItem,
+} from "@trussworks/react-uswds";
+
+import { USWDSIcon } from "src/components/USWDSIcon";
+
+const ProcessProgress = () => {
+  const t = useTranslations("Process.progress");
+  const messages = useMessages() as unknown as IntlMessages;
+
+  const keys = Object.keys(messages.Process.progress.list);
+
+  return (
+    <GridContainer className="grid-container padding-y-4 tablet:padding-y-3 desktop-lg:padding-y-6 padding-top-0 tablet:padding-top-0 desktop-lg:padding-top-0 border-bottom-2px border-base-light">
+      <h2 className="margin-bottom-0 tablet-lg:font-sans-l desktop-lg:font-sans-xl margin-top-0">
+        {t("title")}
+      </h2>
+      <Grid row gap>
+        <Grid className="tablet:grid-col">
+          <IconList className="usa-icon-list--size-md grid-row">
+            {keys.map((key) => {
+              const title = t(`list.${key}.title`);
+              const content = t.rich(`list.${key}.content`, {
+                p: (chunks) => (
+                  <p className="font-sans-md line-height-sans-4 desktop-lg:line-height-sans-6">
+                    {chunks}
+                  </p>
+                ),
+                linkWikiInvolved: (chunks) => (
+                  <Link
```

Comment:
looks like the Figma has a bolder weight for these links. Is that something we need to change or is that something that will change with the branding work? @snyder-ux